### PR TITLE
Visit Model and Storage implementation

### PR DESCRIPTION
**Pull request by jovantanyk**
Initial implementation for Visit Model and Storage.
Todo before merge:
- [x] Fix JsonSerializableAddressBookTest failing 
- [x] Add test cases for Visit and JsonAdaptedVisit

AddVisit command and commandparser in a seperate issue

### CODE REVIEW

**Comment by jovantanyk** [comment]
Okay im done please help review thanks @NatLeong

File: src/main/java/seedu/address/logic/Messages.java [hunk 1]
```diff
@@ -66,4 +69,17 @@ public static String format(Person person) {
         return builder.toString();
     }
 
+    /**
+     * Formats the {@code visit} for display to the user.
+     */
+    public static String format(Visit visit) {
```
Comment (NatLeong) [inline src/main/java/seedu/address/logic/Messages.java:75]: Good idea of overloading format method, will be useful later on!

File: src/main/java/seedu/address/logic/parser/AddVisitCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ */
+public class AddVisitCommandParser implements Parser<AddVisitCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddCommand
+     * and returns an AddCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    //TODO test cases
+    public AddVisitCommand parse(String args) throws ParseException {
```
Comment (NatLeong) [inline src/main/java/seedu/address/logic/parser/AddVisitCommandParser.java:33]: To update in UG: How arguments do not need to be in sequence in format

**Review by NatLeong — COMMENTED** [review]
All looks good. LGTM

**Pull request merged**